pennylane/decomposition/decomposition_graph.py

```diff
@@ -46,8 +46,11 @@
     AdjointDecomp,
     adjoint_adjoint_decomp,
     adjoint_controlled_decomp,
+    adjoint_pow_decomp,
     has_adjoint_decomp,
     has_adjoint_ops,
+    pow_decomp,
+    pow_pow_decomp,
```

Comment:
Do we have a pow_ctrl_decomp interaction? :sweat_smile:

Reply:
No, we plan to add full support to `Pow` in the next quarter, this PR is supposed to be just a naive implementation, providing the most basic support for `Pow`